data-raw/country.codes.R

```diff
@@ -28,7 +28,7 @@ loadPkgData <- function(fname)
              " to create this data set.")
 
     }
-    load(data.file, .GlobalEnv)
+    readRDS(data.file, .GlobalEnv)
```

Comment:
The second argument to `readRDS` is `refhook` which is described as ` a hook function for handling reference objects.` in the documentation. Admittedly I don't know what the intent of `refhook` is but I'd be surprised if it works if you pass the global environment there.

Reply:
Removed the argument, build still works. Thanks for spotting that.